metrics-runtime/src/registry/metric.rs

```diff
@@ -25,19 +26,15 @@ impl MetricRegistry {
         }
     }
 
-    pub fn get_value_handle(&self, identifier: MetricIdentifier) -> MetricValue {
+    pub fn get_value_handle(&self, id: Identifier) -> ValueHandle {
```

Comment:
The rust convention is to remove the `get_` entirely. So you would want `value_handle` here as your method name.

Reply:
I feel like it would actually be the other way around?  If you look at `Vec` or `HashMap`, they both have getters called `get`.

Reply:
Or, alternative, maybe `get_or_register`, since it's technically do more of a HashMap entry API sort of operation than purely a get?

Reply:
From what I understand, your only use `get` when your data only has one obvious thing that your could get. In the case of a `Vec`, that would be the data stored in the vec. I'll find a concrete example for the std lib.

Reply:
But I get what you are saying. Using `value_handle` would give the false sense that the operation is inexpensive.

Reply:
`get_or_register` sound pretty good because it sounds like something fishy is going on.

Reply:
Or maybe something that refers to the fact that this is basically a lazy initialization?

Reply:
I changed this to `get_or_register`.